gcf_data_mapper/read.py

```diff
@@ -0,0 +1,36 @@
+import csv
+import json
+from typing import Any, Optional
+
+import click
+
+
+def read_csv(file_path: str) -> list[str]:
+    """Reads a csv file and returns a list of values"""
+    with open(file_path, "r") as file:
+        csv_reader = csv.DictReader(file)
+        data = []
+        for line in csv_reader:
+            data.append(line["country"])
+        return data
+
+
+def read_json(file_path: str) -> dict:
+    """Reads a json file, we will just read the file for now and echo a value"""
+    with open(file_path, "r") as file:
+        data = file.read()
+        return json.loads(data)
+
+
+def read_data_file(file_path: str) -> Optional[dict[str, Any] | list[str]]:
```

Comment:
Missing param and return in docstring, please see [here](https://github.com/climatepolicyradar/navigator-admin-backend/blob/2674536196424175743efe20d72af0cc8cd2942f/app/repository/document.py#L201) for example